polyfill/intersection-observer.js

```diff
@@ -131,8 +131,12 @@ function IntersectionObserver(callback, opt_options) {
     throw new Error('callback must be a function');
   }
 
-  if (options.root && options.root.nodeType != 1) {
-    throw new Error('root must be an Element');
+  if (
+    options.root &&
+    options.root.nodeType != 1 &&
+    options.root.nodeType !== 9
```

Comment:
Let's make both the same: either `!=` or `!==`.

Reply:
made both `!=`, to match the precedent.